package.json

```diff
@@ -34,7 +34,7 @@
     "common-tags": "^1.4.0",
     "ember-cli-babel": "^7.1.3",
     "ember-cli-test-loader": "^2.2.0",
-    "qunit": "~2.7.1"
+    "qunit": "~2.8.0"
```

Comment:
Can you change this to `^2.80` so that we don’t have to bump manually for every minor version?

Reply:
resolved and updated yarn.lock